### DISCUSSION TITLE
Which version of Wireshark fully supports CoAP-TMF decoding?

### DISCUSSION THREAD

**Discussion by jinpeng1989** [openthread/openthread#9918] [Q&A] [answered]
My Wireshark(v4.2.3) can't decode Thread Network Diagnostic TLV well. It cannot decode the format, even cannot recognize some TLVs. My Wireshark is the latest stable release. Which version of Wireshark fully supports TMF decoding? Or are there any plugins available? 
![image](https://github.com/openthread/openthread/assets/97089879/4f685e02-21b8-4e00-8d93-fa213f04b42e)

**Comment by jwhui** [comment, marked as answer]
The TLVs you referenced above are relatively new to the Thread Specification. Dissector support for those TLVs are probably not yet contributed to the Wireshark project.

Note that Wireshark is an open source project. It would be great if someone from the OpenThread community could help contribute dissector support for newly defined Thread TLVs.
